backend/users/urls.py

```diff
@@ -0,0 +1,6 @@
+from django.urls import path
+from .views import UserRegistrationView
+
+urlpatterns = [
+    path('signup/', UserRegistrationView.as_view(), name='user-signup'),
```

Comment:
The pathname signup/ can be specified as register/ if you want to adhere to API terminology or JWT compatibility.

path('register/', UserRegistrationView.as_view(), name='register'),